frontend/src/app/types/dashboard/timeline/enums/task-status.ts

```diff
@@ -0,0 +1,6 @@
+export enum TaskStatus {
+    Completed = 'Completed',
```

Comment:
Do you really need string values? As I see you use it just like enum, so you can list just as:
{
Completed,
Failed,
InProgress,
ToDo
}

Reply:
Fixed